clifford/test/test_g3c_tools.py

```diff
@@ -247,8 +252,9 @@ def test_dual(self):
         up_array = test_array.up()
         I5 = layout.blades['e12345']
 
-        npt.assert_almost_equal((up_array * ConformalMVArray([I5])).value,
-                                ConformalMVArray([i * I5 for i in up_array]).value)
+        npt.assert_allclose((up_array * ConformalMVArray([I5])).value,
+                ConformalMVArray([i * I5 for i in up_array]).value,
+                            rtol=RTOL_DEFAULT, atol=ATOL_DEFAULT)
```

Comment:
flake8 is failing due to the weird indent.
```suggestion
        npt.assert_allclose(
                (up_array * ConformalMVArray([I5])).value,
                ConformalMVArray([i * I5 for i in up_array]).value,
                rtol=RTOL_DEFAULT, atol=ATOL_DEFAULT)
```